backend/src/main/java/capstone/backend/model/dto/order/OrderToSupplierDTO.java

```diff
@@ -2,21 +2,25 @@
 
 
 import capstone.backend.model.dto.contact.SupplierDTO;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.With;
+import lombok.*;
 import lombok.experimental.SuperBuilder;
 
+import java.util.List;
+
 
 @EqualsAndHashCode(callSuper = true)
 @Data
 @SuperBuilder
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
is Lombok able to call super?

Reply:
I think I needed to particularly state it here for some reason